app/onboarding/views/final.py

```diff
@@ -0,0 +1,26 @@
+from app.extensions import limiter
+from app.onboarding.base import onboarding_bp
+from app.email_utils import send_test_email_alias
+from flask import render_template, request, flash
+from flask_login import current_user, login_required
+from flask_wtf import FlaskForm
+from wtforms import StringField, validators
+
+
+class SendEmailForm(FlaskForm):
+    email = StringField("Email", validators=[validators.DataRequired()])
+
+
+@onboarding_bp.route("/final", methods=["GET", "POST"])
+@login_required
+@limiter.limit("10/minute")
+def final():
+    form = SendEmailForm(request.form)
+    if form.validate_on_submit():
+        send_test_email_alias(form.email.data, current_user.name)
+        flash("We have sent a test e-mail to your alias", "success")
+
+    return render_template(
+        "onboarding/final.html",
```

Comment:
I think the page misses some elements
- an indication for users to click on the SL icon to create a new alias
- an indication that user can also right click to create a new alias
- a link at bottom to the web dashboard

<img width="966" alt="image" src="https://user-images.githubusercontent.com/3167780/168232755-97db371f-1cd5-4590-a2df-37b99d5ed6ad.png">

Reply:
I've added the link at the bottom to the web dashboard. I'll try to finish the indicators for the users to click either on the extension icon or to use the right click